Tests/Requests/View_RequestScreen_Approve_Button_Click.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using Controller.DbControllers;
+using Model.Database.Contexts;
+using Model.DbModels;
+using NUnit.Framework;
+using View.Screens;
+
+namespace Tests.Requests
+{
+    [TestFixture]
+    public class View_RequestScreen_Approve_Button_Click
+    {
+        private MockDatabaseContext context;
+        private RequestController requestController;
+        private Request testRequest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            context = new MockDatabaseContext();
+            requestController = RequestController.Create(context);
+            testRequest = new Request()
+                {ID = 1, UserID = 1, ArtistName = "TestArtist", ArtistReason = "Just because I'm testing"};
+            requestController.CreateItem(testRequest);
+        }
+
+        [Test]
+        public void ApproveUserAsArtist()
+        {
+            var requestScreen = new RequestScreen();
+            requestScreen.ApproveUser(testRequest.ID, context);
+
+            Assert.False(RequestController.Create(context).GetList().Contains(testRequest));
```

Comment:
Waarom gebruik je hier niet `requestController` ipv een hele nieuwe aanmaken?

Reply:
En waarom kijk je niet gewoon of het `testRequest.ID` in de column staat, ipv eerst de hele lijst ophalen?

Reply:
prima